# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/protocol/http/nsIHttpChannelInternal.idl
# Commit: 2502449c2e24
# Full Hash: 2502449c2e24993d495c886c717e7f9546cd9e17
# Author: Kershaw Chang <kershaw@mozilla.com>
# Date: 2020-08-05 21:44:54
# Regressor Bug: 1497249
# File Overlap Count: 4
# Description:
#   Bug 1497249 - P2: ipdl for nsIWebSocketConnection r=michal
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D30624
# ==============================================================================

diff -r 432c7e9dc19f -r 2502449c2e24 netwerk/protocol/http/nsIHttpChannelInternal.idl
--- a/netwerk/protocol/http/nsIHttpChannelInternal.idl	Mon Aug 03 10:03:08 2020 +0000
+++ b/netwerk/protocol/http/nsIHttpChannelInternal.idl	Wed Aug 05 15:17:38 2020 +0000
@@ -27,6 +27,7 @@
 interface nsISecurityConsoleMessage;
 interface nsISocketTransport;
 interface nsIURI;
+interface nsIWebSocketConnection;
 
 /**
  * The callback interface for nsIHttpChannelInternal::HTTPUpgrade()
@@ -40,6 +41,8 @@
                                          in nsIAsyncOutputStream aSocketOut);
 
     [must_use] void onUpgradeFailed(in nsresult aErrorCode);
+
+    [must_use] void onWebSocketConnectionAvailable(in nsIWebSocketConnection aConnection);
 };
 
 /**